### ISSUE TITLE
BLT Sync Commands Don't Respect -y

### ISSUE THREAD

**Issue by mikemadison13** [acquia/blt#1853]
My system information:
* Operating system type: Mac OS
* Operating system version: 10.11.6
* BLT version: 8.9.0

When I run this command:
```
blt sync:refresh -y
blt sync:db -y
```

I get the following output:
```
blt sync:db -y
Executing command blt sync:db -y --define drush.alias=self inside of Drupal VM...
[warning] The xDebug extension is loaded. This will significantly decrease performance.
> setup:settings
[Acquia\Blt\Robo\Tasks\DrushTask] Running /var/www/<project>/vendor/bin/drush cache-clear drush --uri=default && /var/www/<project>/vendor/bin/drush sql-drop --uri=default && /var/www/<project>/vendor/bin/drush sql-sync @<project>.test @<project>.local --structure-tables-key=lightweight --create-db --sanitize --uri=default && /var/www/<project>/vendor/bin/drush cache-clear drush --uri=default in /var/www/<project>/docroot
'drush' cache was cleared.                                                                                                        [success]
Do you really want to drop all tables in the database drupal? (y/n):
```

And I expected this to happen:
I would expect the -y to automatically clear each question during the sync for dropping the db, but it does not.

**Comment by dpagini** [comment]
Looks like a dupe of #1841. Slightly different request I think, but same root problem.

**Comment by mikemadison13** [comment]
@dpagini yeah it does look like a dupe, i'm flexible in how this is approached. I just remember @grasmash saying that it "should" accept a -y (and blt itself indicates that). So i'm not sure if we want to fix both of this issues and assume try AND accept a -y / -n or not.

**Comment by grasmash** [comment]
I consider this addressed by #1841.
